frontend/src/login_helper.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Fetch login status from backend. Return Object containing login status and login or logout URL.
+ */
+export async function getLoginStatus() {
+  const loginResponse = await fetch("/api/v1/login").then(response => response.json());
```

Comment:
btw recently found out its bad practice to mix 'await' and 'then' calls.

https://docs.google.com/document/d/1nzCdpnjwqMvESrg-P6b1r_0nNaxhGf5AvrD-Yv_mXgM/edit

but also there are some good pointers there on error handling and such try/catch/finally structure as well some async await possible bugs that might prop up that is good to understand anyway.